### DISCUSSION TITLE
Layout manager showing elements that are not showed in the demo version

### DISCUSSION THREAD

**Discussion by jayrodrigues** [GrapesJS/grapesjs#4862] [Q&A]
Hello!

The layout manager shows Meta, Text, and Link elements. On the demo version on the grapejs site - these elements are not shown. Can you direct me to where the setting for this is?

And if a column is added, in the layout manager, it shows as Box. But on the main demo, it shows up as Row. 

Look forward to getting your input

**Comment by artf** [comment]
There are different plugins running on the demo, you can check the source here: https://github.com/artf/grapesjs/blob/gh-pages/demo.html

**Comment by jayrodrigues** [comment]
Hi @artf, thanks for the response. I checked the source but even though I have similar plugins.

I feel like I am missing some attributes/flags in the code where these basic elements are not displayed. 

I am attaching a screenshot to help clarify what I am referring to.

You will see the body area has no elements but on the layout manager, it shows the Meta, Title and Text elements. 

Are you able to point me to what attributes may need to be modified to get the layout manager not to show those elements?

Thank you
![image](https://user-images.githubusercontent.com/13894376/218148848-70d44539-2386-4402-ab57-5309f0b0cd53.png)

**Comment by jayrodrigues** [comment]
Hello @artf - do you happen to have an answer on this for me?

**Reply by artf** [reply]
In order to help you properly I'd need to see a  reproducible demo of your use case.
